fire_atlas_editor/lib/widgets/container.dart

```diff
@@ -1,12 +1,12 @@
 import 'package:flutter/material.dart';
 
 class FContainer extends StatelessWidget {
```

Comment:
What does the F stand for?

Reply:
Fire Atlas, I just wanted to avoid name clashing with the framework.

Totally open to suggestions on a better prefix btw hahhhaha

Reply:
I would write it all out `FireAtlasContainer`, or handle the clashes when it is used by hiding clashing classes or by using named imports. :)

Reply:
I am planning a follow up to refactor a bunch of things, I will this to the list and do on a follow up